src/Sarif/NotYetAutoGenerated/AddressComparer.cs

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft.  All Rights Reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    /// <summary>
+    /// Defines methods to support the comparison of objects of type Address for sorting.
+    /// </summary>
+    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "1.1.4.0")]
+    internal sealed class AddressComparer : IComparer<Address>
```

Comment:
>AddressComparer [](http://example.com/codeflow?start=26&length=15)

This class is need because the NotYetAutoGenerated version of Address replaced couple properties of int type to int?. The nullable type is not supported yet for Comparer code generator.